lib/python/vdm/tests/server/database_test.py

```diff
@@ -172,7 +172,7 @@ def test_request_with_id_member(self):
         response = requests.put(url, json={'name': 'test123', 'id': last_db_id})
         value = response.json()
         if response.status_code == 200:
-            self.assertEqual(value['status'], 1)
+            self.assertEqual(value['status'], 200)
```

Comment:
Why are many of these tests checking if the status_code is 200 and then asserting that it is 200. That seems pointless.